app/models/form1095_b.rb

```diff
@@ -89,16 +95,24 @@ def generate_pdf(pdftk, tmp_file)
         'topmostSubform[0].Page1[0].Part1Contents[0].Line1[0].f1_02[0]': data[:middle_name],
         'topmostSubform[0].Page1[0].Part1Contents[0].Line1[0].f1_03[0]': data[:last_name],
         'topmostSubform[0].Page1[0].Part1Contents[0].f1_04[0]': data[:last_4_ssn] || '',
-        'topmostSubform[0].Page1[0].Part1Contents[0].f1_05[0]': data[:last_4_ssn] ? '' : data[:birth_date],
+        'topmostSubform[0].Page1[0].Part1Contents[0].f1_05[0]': ssn_or_birthdate,
         'topmostSubform[0].Page1[0].Part1Contents[0].f1_06[0]': data[:address],
         'topmostSubform[0].Page1[0].Part1Contents[0].f1_07[0]': data[:city],
         'topmostSubform[0].Page1[0].Part1Contents[0].f1_08[0]': data[:state] || data[:province],
         'topmostSubform[0].Page1[0].Part1Contents[0].f1_09[0]': country_and_zip,
```

Comment:
Added the VA contact info rather than changing the template manually. More robust for the future.